sbin/setkey/setkey.8

```diff
@@ -686,18 +691,18 @@ include authentication and should not be
 paired with a separate authentication algorithm via
 .Fl A .
 .Ss Compression Algorithms
-The following compression algorithms can be used
+The following compression algorithm can be used
```

Comment:
The previous text was correct: `algorithms`, since it's plural.

Reply:
But there's only one option?

Reply:
I apologize, I looked at this way too quickly.

It's a weird subsection.  Commit e6dede145616ed8f98c629c23a2ba206b812c921 added a `.Ss` with a plural, and shortened the previous text (and kept the existing plural in that text).  The very first commit 9a4365d0e0833374e893c519639bde71756aa104 had two compression algorithms, `deflate` and `lzs`.

Commit cf43a05493795bba68b85dc4c43219604b180755 removed it from the man page, but not from the code (as far as I can see).  Even today, `token.l` still contains `lzs`... along with `oui` (which I haven't heard of).

I am suspicious that the man page is inaccurate, but this is far from my area of expertise, so I withdraw my objection to this change.

Reply:
Wow that adds a lot of context, thank you! Hopefully someone who uses this will come chime in. If they're in the code and work, I'd love to get them in here.